controller/test-files/event/list/ok.bool_list.res.payload.golden.json

```diff
@@ -0,0 +1,74 @@
+{
+  "data": [
+    {
+      "attributes": {
+        "name": "bool_list",
+        "newValue": [
```

Comment:
@kwk if https://github.com/fabric8-services/fabric8-wit/pull/2213/files#r208150176 is correct, then why `oldValue` is missing for `bool_list`? `bool_list` isn't a relational field.

Reply:
Good point. Let me have a look...

Reply:
I think we're fine omitting the field entirely if the value is null, right @jarifibrahim ? Then I can make that change accordingly.

Reply:
Yes. We should omit the `oldValue` field if the value is null.